DOCSP-16785 the the typo (#3297)
mongodb/docs-realm: temp/dotnet/manage-users/custom-user-data.txt

Custom User Data - .NET SDK
===========================

.. meta:: 

>   :description: Learn how to create custom user data using Atlas App Services in .NET

Reviewer: ```suggestion
  :description: Learn how to create custom user data using the Atlas Device SDK for .NET. 
```
